tests/unit/components/Navigator/QuickNavigationModal.spec.js

```diff
@@ -310,4 +310,26 @@ describe('QuickNavigationModal', () => {
     });
     expect(wrapper.vm.processedUserInput).toBe('barfoo');
   });
+  it('it removes filtered symbols with duplicate paths', () => {
+    const symbolsWithRepeatedPaths = [
```

Comment:
Minor comment: you could add another obj with a different path to the `symbolsWithRepeatedPaths` array so you can make sure that you are not returning duplicate ones instead of filtering them out.

Reply:
Resolved https://github.com/apple/swift-docc-render/pull/496/commits/aa5168129b19e9b4f1643cf99059659b4fb42247